frame/evm/src/runner/stack.rs

```diff
@@ -80,8 +80,7 @@ where
 		precompiles: &'precompiles T::PrecompilesType,
 		is_transactional: bool,
 		weight_limit: Option<Weight>,
-		proof_size_base_cost: Option<u64>,
-		measured_proof_size_before: u64,
+		proof_size_pre_execution: u64,
```

Comment:
We should use an `Option` here, it will also simplify the code a bit.
```suggestion
		proof_size_pre_execution: Option<u64>,
```